src/emr_cluster.go

```diff
@@ -263,6 +281,37 @@ func (ec EmrCluster) GetInstanceGroups() []*emr.InstanceGroupConfig {
 	return instanceGroups
 }
 
+// GetEbsConfiguration turns a EbsConfigurationRecord into an emr.EbsConfiguration
+func GetEbsConfiguration(c *EbsConfigurationRecord) *emr.EbsConfiguration {
+	configs := c.EbsBlockDeviceConfigs
+
+	var emrConfigsArr []*emr.EbsBlockDeviceConfig
+
+	if configs != nil && len(configs) > 0 {
+		emrConfigsArr = make([]*emr.EbsBlockDeviceConfig, len(configs))
+
+		for i, config := range configs {
+			emrVolumeSpec := emr.VolumeSpecification{
+				Iops:       aws.Int64(config.VolumeSpecification.Iops),
+				SizeInGB:   aws.Int64(config.VolumeSpecification.SizeInGB),
+				VolumeType: aws.String(config.VolumeSpecification.VolumeType),
```

Comment:
Apparently simple optional types are not supported by avro-go :+1:, `Iops` is always nil no matter if it's present.

As a result, we can't enforce this at the schema level on our side and this isn't enforced by aws either.

I'll investigate how default values are handled.